server/form-pages/assess/reviewApplication/sufficientInformation/sufficientInformationSent.ts

```diff
@@ -16,18 +17,29 @@ export default class SufficientInformationSent implements TasklistPage {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   constructor(public readonly body: Record<string, unknown>) {}
 
-  user: ApprovedPremisesUser
+  userName: string
```

Comment:
I think it would be clearer if we used the types from the Body type in `confirmYourDetails.ts` here. 
EG `Body['username']`. We could alias `Body` to be ConfirmYourDetailsBody for clarity if you think that would be useful.